system/conf/web.conf

```diff
@@ -5,6 +5,6 @@
 // Path to the PHP CGI executable
 // If the file doesn't exist, and the configured path is in user/tools/,
 // PHP is downloaded automatically on Windows.
-php_cgi_bin: user/tools/php/php-cgi.exe
+php_cgi_exe: /usr/bin/php-cgi
```

Comment:
You seem to have reverted a recent change here, where the name of the option was fixed. It was supposed to be `_bin`. Additionally, it would be better if, instead of changing default values in conf files in the system folder, you would dynamically make your changes in the user folder for Docker.